.github/workflows/build-on-ubuntu.yml

```diff
@@ -22,23 +22,13 @@ jobs:
         run: ./gradlew build --stacktrace
 
       # See: https://github.com/marketplace/actions/junit-report-action
-      - name: Publish Unit Test Report
+      - name: Publish Test Report
         uses: mikepenz/action-junit-report@v3.5.2
         if: always() # always run even if the previous step fails
         with:
-          check_name: Regular Unit Tests
           report_paths: '**/build/test-results/test/TEST-*.xml'
           require_tests: true # will fail workflow if test reports not found
 
-      # See: https://github.com/marketplace/actions/junit-report-action
```

Comment:
Can we revert these changes please? Separation of test reports was needed some time ago when investigating build failures at CI.